gsms.py

```diff
@@ -180,12 +184,36 @@ def main() -> None:
 
                 if not app_exists:
                     count += 1
+                    if shortcut.work_dir.endswith(os.sep): # remove final path separator but only if it exists
```

Comment:
change from `if` to `while`... this will handle cases where there is a double path separator (probably rare, but I'm sure it will happen)

Reply:
has been changed to `while` good idea as it might happen.